docs/Fault-Tolerant-Tachyon-Cluster.md

```diff
@@ -3,12 +3,28 @@ layout: global
 title: Fault Tolerant Tachyon Cluster
 ---
 
+# Fault Tolerance
+
+Fault Tolerance in Tachyon is based upon a multi-master approach where multiple master processes are
+run.  One of these processes is elected the leader and is used by all workers and clients as the primary 
+point of contact.  The other masters act as a standby using the shared journal to ensure that they maintain the same
```

Comment:
@calvinjia Line lengths have been reduced to be in line with rest of document

@uronce-cc Reworded to explicitly say that it is the filesystem metadata that is kept in sync